utils/migrate_latency_results_to_argus.py

```diff
@@ -0,0 +1,114 @@
+
+from sdcm.argus_results import send_result_to_argus
+from sdcm.keystore import KeyStore
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def migrate(creds, job_name, days=7, index_name="performancestatsv2", dry_run=True):  # pylint: disable=too-many-locals
+    ks = KeyStore()
+    es_conf = ks.get_elasticsearch_token()
+    elastic_search = Elasticsearch(**es_conf)
+
+    query = {
+        "query": {
+            "bool": {
+                "must": [
+                    {
+                        "range": {
+                            "test_details.start_time": {
+                                "gte": (datetime.datetime.utcnow() - datetime.timedelta(days=days)).timestamp(),
+                                "lte": datetime.datetime.utcnow().timestamp(),
+                                "boost": 2.0
+                            }
+                        }
+                    },
+                    {
+                        "term": {
+                            "test_details.job_name.keyword": job_name
+                        }
+                    }
+                ]
+            }
+        },
+        "size": 10000
+    }
+
+    res = elastic_search.search(index=index_name, body=query)
+    for hit in res['hits']['hits']:
+        document_id = hit['_id']
+        print(f"Document ID: {document_id}")
+        workload = hit['_source']['test_details']['sub_type']
+        client = ArgusSCTClient(document_id, auth_token=creds["token"], base_url=creds["baseUrl"])
+        try:
+            latency_during_ops = hit['_source']['latency_during_ops']
+        except KeyError:
+            print(f"Document {document_id} does not have latency_during_ops key")
+            continue
+        for operation in latency_during_ops:
+            if operation == "summary":
+                continue
+            if operation == "Steady State":
+                result = latency_during_ops[operation]
+                description = "Latencies without any operation running"
+                if dry_run:
+                    print(f"Would send {operation} - {workload} - latencies - cycle 0 to Argus")
+                    continue
+                try:
+                    send_result_to_argus(argus_client=client, workload=workload, name=operation,
+                                         description=description, cycle=0, result=result)
+                except argus.client.base.ArgusClientError:
+                    print(
+                        f"Failed to send {operation} - {workload} - latencies to Argus: {hit['_source']['test_details']['job_url']}")
+                continue
+            for idx, cycle in enumerate(latency_during_ops[operation]["cycles"], start=1):
+                if dry_run:
+                    print(f"Would send {operation} - {workload} - latencies - cycle {idx} to Argus")
+                    continue
+                try:
+                    send_result_to_argus(
```

Comment:
do we handle duplicates report ? i.e. we'll want to consider running with production argus data ?

not critical to handle it, but just to know if it would be an option or not

Reply:
In case of duplication it will override the data.
Still one more thing to fix, as I want to upload it in proper order (to fix issue with wrong order of tables in Argus).